docs/docsite/rst/scenario_guides/guide_docker.rst

```diff
@@ -43,12 +43,24 @@ Requirements
 ------------
 
 Using the docker modules requires having `docker-py <https://docker-py.readthedocs.io/en/stable/>`_
-installed on the host running Ansible. You will need to have >= 1.7.0 installed.
+installed on the host running Ansible. You will need to have >= 1.7.0 installed. For Python 2.7 or
+Python 3, you can install it as follows:
+
+.. code-block:: bash
+
+    $ pip install docker
+
+For Python 2.6, you need a version before 2.0. For these versions, the module was called ``docker-py``,
+so you need to install it as follows:
 
 .. code-block:: bash
 
     $ pip install 'docker-py>=1.7.0'
 
+Please note that only one of ``docker`` and ``docker-py`` must be installed. Installing both will
+result in a broken installation, which can be recovered by first uninstalling both and then re-installing
```

Comment:
What behavior would the user see if they had this broken installation? Error message? Failure?

Reply:
Yeah, good solution. If you know what text Ansible will spit out, include that too.

Reply:
I've changed it to `Please note that only one of ``docker`` and ``docker-py`` must be installed. Installing both will result in a broken installation. If this happens, Ansible will detect it and inform you about it::` followed by the error message.

Reply:
d'oh, that **was** the error text - thanks for including it

Reply:
Yep; it got a bit longer over time, but hopefully it will prevent some more issues and complaints from people who think they will have `docker` still installed (after removing `docker-py`) and then getting strange error messages... ;)